src/examples/nintendoswitch/debug-hello.go

```diff
@@ -0,0 +1,17 @@
+// This program is a minimal Nintendo Switch program that just outputs messages to
+// emulator console
+package main
+
+import (
+	"runtime"
+	"time"
+)
+
+func main() {
+	runtime.NxOutputString("Hello world!")
```

Comment:
In principle, `examples/serial` should work as well. Would that work as a smoke test? It would reduce the amount of code specific to the Switch and proves that `println` works correctly.

Reply:
@racerxdl also this comment.

Reply:
Just tested, it does indeed work because of `putchar` implementation. I removed this nintendo switch example and changed the smoke test to build examples/serial.